functional-samples/tutorial.websockets/README.md

```diff
@@ -0,0 +1,12 @@
+# Chrome Extension WebSocket Example
+
+This example demonstrates how to use WebSockets in a MV3 Chrome Extension starting with Chrome version M116.
```

Comment:
Can we say that from Chrome 116 the service worker should stay alive?

Reply:
done